### PR TITLE
UI: input components

### LINKED ISSUE #63: Apply styles on Remark component.
This is mainly the same design as #64  but without no receiver field.

### PR DESCRIPTION
**Pull request by goldsteinsveta**
This is a first iteration. Closes #63, #64, #65
Screenshot 2021-05-03 at 19 04 33

### CODE REVIEW

**Comment by goldsteinsveta** [comment]
I would pixel perfect later

File: src/components/Remark.tsx [hunk 1]
```diff
@@ -46,23 +45,15 @@ const Remark = ({ className }: Props) => {
...
-        <Button variant="contained" disabled={isButtonDisabled()} onClick={sendLaneMessage}>
-          Send Remark
-        </Button>
-        <p>{estimatedFee && `Estimated source Fee: ${estimatedFee}`}</p>
-      </Container>
+      <TextField onChange={onChange} value={remarkInput} variant="outlined" fullWidth />
+      <ButtonSubmit disabled={isButtonDisabled()} onClick={sendLaneMessage}>
+        Send bridge remark from {sourceChainDetails.sourceChain} to {targetChainDetails.targetChain}
+      </ButtonSubmit>
+      <p>{estimatedFee && `Estimated source Fee: ${estimatedFee}`}</p>
```
Comment (wirednkod) [inline src/components/Remark.tsx:54]: add <p> after the "estiamtedFee" check. There is no reason to have an empty tag if estimatedFee does not exist

File: src/components/Transfer.tsx [hunk 2]
```diff
@@ -14,29 +14,39 @@
...
+      }
+    }
   }
 }));
 
 const Transfer = () => {
   const classes = useStyles();
   const [isRunning, setIsRunning] = useState(false);
-  const [transferInput, setTransferInput] = useState('0');
+  const [transferInput, setTransferInput] = useState('');
```
Comment (wirednkod) [inline src/components/Transfer.tsx:48]: Always add the type of the state:
```
const [transferInput, setTransferInput] = useState<string>('');
```

File: src/screens/Main.tsx [hunk 3]
```diff
@@ -53,12 +54,6 @@ const MenuContents = [
     title: 'Custom Call',
     isEnabled: true,
     component: <CustomCall />
-  },
-  {
```
Comment (wirednkod) [inline src/screens/Main.tsx:57]: Oh no! why would you do that? This is used in a different PR and should be part of the dropdown menu

**Review by wirednkod — CHANGES_REQUESTED** [review]

File: src/screens/Main.tsx (hunk 3 above)
Comment (goldsteinsveta) [reply src/screens/Main.tsx:57]: Which PR? Do we really need it? Doesn't seem to be the case for me. Should I revert changes?

Comment (wirednkod) [reply src/screens/Main.tsx:57]: > Which PR? Do we really need it? Doesn't seem to be the case for me. Should I revert changes?

Hm... Nah - Just leave it and we'll fix it after merge

**Review by wirednkod — APPROVED** [review]

**Review by tomusdrw — APPROVED** [review]

**Pull request merged**